scripts/compose/grafana/provisioning/dashboards/dashboard.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
```

Comment:
Commit message: Something seems to be wrong in "to view the Loki logs to the Grafana configuration". Maybe "to add..."?